### DISCUSSION TITLE
How do tag filter works ?

### DISCUSSION THREAD

**Discussion by Darkun92** [marcusolsson/obsidian-projects#844] [Help]
Hello

For example, I have several notes tagged as #AMOA. They well appear in my project.
When I want to use the tag filter, I try to write #AMOA but the selector stays empty. When I write AMOA, it keeps the value but the filter does no work.
I must do something wrong but no idea where. Can anybody help ?

**Comment by Acylation** [comment]
"#" is uneccessary for a tags input. And since the tags filter should accept an array of tags, you need to press enter or click outside of the input to accept each value. Once you're done, you can savely close the filter pop-over and see the filter taking effect.

![image](https://github.com/marcusolsson/obsidian-projects/assets/73122375/3449e3e2-73de-4d1a-93a4-7a3efda05c44)

**Reply by Darkun92** [reply]
Thks, works well !
